### DISCUSSION TITLE
Are you using freezed in your products?

### DISCUSSION THREAD

**Discussion by wamynobe** [rrousselGit/freezed#1052] [General]
Hi community, our team is going to start a new project with Flutter. First of all, I suggested to use freezed because of its convenience but some members worried that too much generated classes by freezed may make our app slower (they believed that each abstract layer may slow down our app couple ms). If you guys are using freezed in your product, please share your thought. Thank for reading!

**Comment by mono0926** [comment]
Yes, I'm using freezed. I feel that the benefits far outweigh the drawbacks.
